### PR TITLE
New docker readme

### PR DESCRIPTION
**Pull request by LeandroGuillen**
The docker readme file has undergone substantial changes in structure. Hopefully is more clear how to use the different methods now.

Comments welcome, especially for section names :)

### CODE REVIEW

File: docker/README.md [hunk 1]
```diff
@@ -34,58 +33,112 @@ If you do not have or want to download the Orion repository, you can create a fi
 	  ports:
 	    - "1026:1026"
 	  command: -dbhost mongo
+```
+> Uncomment the `--smallfiles` line if you host doesn't have too much free space.
```
Comment (fgalan) [inline docker/README.md]: This should be aligned with the .yml file we provide after changes in PR https://github.com/telefonicaid/fiware-orion/pull/1395, which uses (without comment):

`command: --smallfiles --nojournal`

The rationale is that it is better to use the "safest" configuration to run in every system (e.g. the ones which small disk space of slow). It doesn't hurt to newbie users and advanced users would know that this is about and can remove that commands if they don't want to use it.

This text could be used to explain a little bit what they are (maybe with a better wording):

"Regarding --smallfiles, you can ommit it if your hosting machine have enough disk space. Regarding --nojournal it is not recommened for production, but it speeds up mongo container start up and avoid some "race conditions" problems if Orion containers is faster and doesn't find the DB up and ready".

File: docker/README.md [hunk 2]
```diff
@@ -34,58 +33,112 @@ If you do not have or want to download the Orion repository, you can create a fi
...
+3. Using the command-line and within the directory you created type: `sudo docker-compose up`.
```
Comment (fgalan) [inline docker/README.md]: Check rendering... it seems that the bullet number 3 is put alongside the "grey part".

File: docker/README.md (hunk 1 above)
Comment (LeandroGuillen) [inline docker/README.md]: Fixed in 4fc7c55

File: docker/README.md (hunk 2 above)
Comment (LeandroGuillen) [inline docker/README.md]: It's funny how vanilla markdown will render something fine while github will not.

Fixed in 4fc7c55

**Comment by LeandroGuillen** [comment]
@dmoranj Take a look also!

**Comment by fgalan** [comment]
LGTM (passing the ball to @dmoranj for final LGTM + merge + delete)

File: docker/README.md [hunk 3]
```diff
@@ -1,91 +1,145 @@
...
-whose name is `mongodb`. In case you need one type `sudo docker run --name mongodb -d mongo:2.6`.
-* In `-p 1026:1026` the first value represents the port to listen on localhost. If you want to run a second context broker
+If you want to know more about images and the building process you can find it in [Docker's documentation](https://docs.docker.com/userguide/dockerimages/).
+
+## 4. Other info
+
+Things to keep in mind while working with docker containers and Orion Context Broker.
+
+### 4.1 Data persistence
+Everything you do with Orion Context Broker when dockerized is non-persistent. *You will lose all your data* if you turn off the MongoDB container. This happens with either this method or the manual one.
```
Comment (dmoranj) [inline docker/README.md]: It's not clear what method you are referring to in the following sentence "This happens with either this method or the manual one".

Comment (LeandroGuillen) [inline docker/README.md]: Good catch. It was actually a left over from the previous version.

Fixed in 830e21d

**Comment by dmoranj** [comment]
LGTM

**Pull request merged**